coralproject/talk
# [CORL-233] Create a user drawer with comment streams

## What does this PR do?

Adds a user drawer to the moderation area when you click on a moderation card username.

## How do I test this PR?

- Go to moderation area
- Find a moderation card
- Click on the user name
- See the user drawer appear with user details and comment streams
- Should have an approve and a reject comment stream
